tfx/experimental/pipeline_testing/dummy_chicago_taxi_pipeline.py

```diff
@@ -0,0 +1,195 @@
+# Lint as: python2, python3
```

Comment:
Let's make a new directory for the chicago_taxi_pipeline, because this is an example, and we will move this to tfx/examples/chicago_taxi_pipeline later.

How about "tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/taxi_pipeline_regression_test.py" ?
Also, it would be better to split this PR to the testing framework part and examples part. 
Let's make another PR for this.